<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>605</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fname">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FastName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LastName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lname">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="user">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UeserName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>250</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>320</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="terms_checkBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Creating your account and you acception </string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="face_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up Using Facebook!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gmail_Button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>470</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign up Using Gmail</string>
   </property>
  </widget>
  <zorder>password</zorder>
  <zorder>submit</zorder>
  <zorder>label</zorder>
  <zorder>fname</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>lname</zorder>
  <zorder>user</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>email</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>confirm</zorder>
  <zorder>checkBox</zorder>
  <zorder>face_Button</zorder>
  <zorder>gmail_Button</zorder>
 </widget>
 <resources>
  <include location="new_img.qrc"/>
 </resources>
 <connections/>
</ui>
